reana_server/scheduler.py

```diff
@@ -43,7 +43,13 @@ def check_memory_availability(workflow_min_job_memory):
     if not nodes:
         return True
     max_node_available_memory = max(nodes)
-    return max_node_available_memory >= workflow_min_job_memory
+    available = max_node_available_memory >= workflow_min_job_memory
+    if not available:
+        logging.info(
+            f"Workflow could not be started due to low memory. Available memory: "
```

Comment:
Cannot start workflow 60515eff-33ca-468a-a55c-6cf72d72af5d because it requires 12 GiB memory whilst only 8 GiB is available.